frontend/src/api/services/statusService.ts

```diff
@@ -0,0 +1,54 @@
+import { isDevServer } from "./util";
+
+/**
+ * Status service allows for a simple status checking mechanism.
+ */
+const pollStatus = async () => {
+    const abort = new AbortController();
+    const healthz = isDevServer() ? "//127.0.0.1:8000/healthz" : "/healthz";
```

Comment:
this is on purpose, `//` indicates to use whichever url scheme is used to load the page